### PR TITLE
Translate quick-start.md in french

### PR DESCRIPTION
**Pull request by nclsppr**

### CODE REVIEW

File: docs/docs/quick-start.md
Comment (mindsers) [inline docs/docs/quick-start.md:13]: « Builder » is not a valid French word. what do you think about « générer ». 
I understand the literal translation « Construire » is ugly but « Générer » seems to match the programming context.

File: docs/docs/quick-start.md
Comment (mindsers) [inline docs/docs/quick-start.md:22]: As all other headers end with a dot, maybe we should put this one back. 
```suggestion
### Créer un nouveau site.
```

File: docs/docs/quick-start.md
Comment (dantereve) [inline docs/docs/quick-start.md:5]:
```suggestion
Cette page de démarrage rapide est dédiée aux développeurs d'un niveau intermédiaire à avancé. Pour une intro plus facile à Gatsby, [dirigez-vous vers notre tutoriel](/tutorial/)!
```

File: docs/docs/quick-start.md
Comment (dantereve) [inline docs/docs/quick-start.md:14]:
```suggestion
**Note**: Cette vidéo utilise `npx`, qui est un outil pour exécuter un package npm sans avoir à l'installer d'abord. Lancer la commande `npx gatsby new` est la même chose que lancer `gatsby new` après avoir installé le gatsby-cli sur votre ordinateur.
```

File: docs/docs/quick-start.md
Comment (dantereve) [inline docs/docs/quick-start.md:40]:
```suggestion
Gatsby va démarrer un environnement de rechargement à chaud accessible par défaut à l'adresse `localhost:8000`.
```

File: docs/docs/quick-start.md
Comment (mindsers) [inline docs/docs/quick-start.md:52]: It’s not exactly « Démarrer » because we loose the idea of « Servir » and « providing » something.

We can’t start a the build but we can start a server which will serve the build to us.

File: docs/docs/quick-start.md
Comment (mindsers) [inline docs/docs/quick-start.md:58]: A little typo here. We forgot the « un » between « démarre » and « serveur ».

Comment (dantereve) [inline docs/docs/quick-start.md:58]:
```suggestion
Gatsby démarre un serveur HTML local pour tester votre site buildé. Rappelez-vous de builder votre site en utilisant `gatsby build` avant d'utiliser cette commande.
```

File: docs/docs/quick-start.md
Comment (mindsers) [inline docs/docs/quick-start.md:40]: We loose some informations here e. g. « development server » is not mentioned. 

More generally I’m wondering if a French dev will easily understand what is a « environnement de rechargement à chaud »? What do you think?

File: docs/docs/quick-start.md
Comment (mindsers) [inline docs/docs/quick-start.md:42]: On the second sentence, it’s not the save that reload the page but the saved changes. 

If the user save a file without changes no reload will be triggered.

**Review by dantereve — CHANGES_REQUESTED** [review]
First PR in Gatsby? 🥳 This is looking good thank you! I've left a few comments that should be addressed before we can merge this, it's only typos, nothing serious.
Now I wonder how we can translate 'build' ... 'compilation' might be ok even if the compilation is just a step of the build ...  Mmh tricky one, what do you think?

**Review by mindsers — CHANGES_REQUESTED** [review]
Thank you for taking the time to contribute 🔥. We really appreciate it!
I'm adding some attention points to the ones of the other maintainers who previously reviewed the PR. There are some open question, so don't hesitate to give us your opinion.

Also, I'm on my phone. I can't add *change suggestions* (not the same editor). Sorry for the inconvenient. 😅

File: docs/docs/quick-start.md
Comment (mindsers) [reply docs/docs/quick-start.md:58]: I resolve this one as it's a duplicate convo (https://github.com/gatsbyjs/gatsby-fr/pull/4#discussion_r341002304)

File: docs/docs/quick-start.md
Comment (gatarelib) [reply docs/docs/quick-start.md:40]: Good question, actually most French Devs use English terminologies like hot-reloading.
Translating such words directly to French would confuse them the more.

File: docs/docs/quick-start.md
Comment (gatarelib) [inline docs/docs/quick-start.md:42]: How about:
Les modifications sauvegardées se rechargeront immédiatement dans le navigateur.

Comment (gatarelib) [reply docs/docs/quick-start.md:42]: I find it easier to say "se rechargeront immédiatement" in the place of "will live reload"
English with buzzwords . . . God knows😂

File: docs/docs/quick-start.md
Comment (gatarelib) [reply docs/docs/quick-start.md:40]: How about "rechargement immédiat" if it must be translated?

**Review by gatarelib — CHANGES_REQUESTED** [review]
"rechargement immédiat" for "Hot reloading"

File: docs/docs/quick-start.md
Comment (mindsers) [reply docs/docs/quick-start.md:42]: After reflexion, it's not even the page that will be reload but only the changes into this page. Full reload of the page comes only when the webpack server don't know how to reload only the changes.

File: docs/docs/quick-start.md
Comment (mindsers) [reply docs/docs/quick-start.md:40]: I like this last version ;)

**Comment by dantereve** [comment]
@nclsppr do you need any help to finish this up?

File: docs/docs/quick-start.md
Comment (Minivera) [reply docs/docs/quick-start.md:13]: The _Office de la langue Français_ in Quebec suggests the word "Construire" or "Assembler" as a translation to "build". I prefer "Générer", but I wanted to point that out 😄. I'm trying to find a similar source for translations on the France side.

http://gdt.oqlf.gouv.qc.ca/ficheOqlf.aspx?Id_Fiche=8352596

File: docs/docs/quick-start.md
Comment (Minivera) [reply docs/docs/quick-start.md:40]: Parcel uses "Remplacement de module à chaud" for its docs https://parceljs.org/hmr.html. Would you think this would be more consistent? From my perspective, both are very clear.

Comment (dantereve) [reply docs/docs/quick-start.md:40]: Both are very clear indeed. I think we'll have to make a decision quickly as this md file is stuck. "Remplacement de module à chaud" is more relevant for webpack and parcel in my opinion.

File: docs/docs/quick-start.md
Comment (dantereve) [reply docs/docs/quick-start.md:13]: Interesting resource, thanks for sharing! We have used "Générer" on almost every page 😱  we'll have to stick with it but we can modify that in the future.

File: docs/docs/quick-start.md
Comment (dantereve) [inline docs/docs/quick-start.md:11]:
```suggestion
  lessonTitle="Démarrage rapide avec Gatsby : Créer, Développer et Générer des sites Gatbsy en ligne de commande"
```

File: docs/docs/quick-start.md
Comment (dantereve) [inline docs/docs/quick-start.md:42]:
```suggestion
Essayez d'éditer les pages JavaScript dans `src/pages`. Les changements sauvegardés vont se recharger en direct dans le navigateur.
```

File: docs/docs/quick-start.md
Comment (dantereve) [inline docs/docs/quick-start.md:44]:
```suggestion
### Créer une version de production.
```
"générer/génération" seems hard to apply here. We could use "version" instead in this context  http://gdt.oqlf.gouv.qc.ca/ficheOqlf.aspx?Id_Fiche=8874716

File: docs/docs/quick-start.md
Comment (dantereve) [inline docs/docs/quick-start.md:50]:
```suggestion
Gatsby va réaliser une version de votre site optimisé pour la production, en générant du HTML statique et des groupements de code JavaScript pour chaque route.
```

File: docs/docs/quick-start.md
Comment (dantereve) [inline docs/docs/quick-start.md:52]:
```suggestion
### Servir la version de production en local.
```

File: docs/docs/quick-start.md
Comment (dantereve) [inline docs/docs/quick-start.md:66]:
```suggestion
Pour plus d'informations sur le Gatbsy CLI, visitez la section [Commandes (Gatsby CLI)](/docs/gatsby-cli/) de la documentation.
```

**Review by dantereve — CHANGES_REQUESTED** [review]

**Comment by mindsers** [comment]
This PR was created in October and since the reviewers asked for changes, we have no news from the author of the pull request. I close the PR as it seems stale. The author can open a new one from the same branch if needed.

**Pull request closed without merging**
